README.md

```diff
@@ -175,14 +175,20 @@ user@host:~$ sudo ldconfig
 ### System Dependencies
 To install the system dependencies, it is possible to use [Homebrew](http://brew.sh/):
 ```
-brew install ace eigen cmake boost tinyxml swig qt5 gsl pkg-config jpeg sqlite readline tinyxml ipopt
+brew install ace eigen cmake boost tinyxml swig qt5 gsl pkg-config jpeg sqlite readline tinyxml ipopt opencv@3
 ```
 
 Since Qt5 is not symlinked in `/usr/local` by default in the homebrew formula, `Qt5_DIR` needs to be properly set to make sure that CMake-based projects are able to find Qt5.
 ```
 export Qt5_DIR=/usr/local/opt/qt5/lib/cmake/Qt5
 ```
 
+The required version of `opencv` is not the latest release. In case other versions are installed, it is required to link `opencv3`.
+```
+brew link --force opencv@3
+```
+Otherwise, it is possible to not use `opencv` by setting `YARP_USES_OPENCV` `OFF`.
```

Comment:
```suggestion
Otherwise, it is possible to not use `opencv` by setting the `YARP_USE_OpenCV` YARP CMake option to  `OFF`. To modify YARP-specific CMAke options, you need to go with the terminal to `build/robotology/YARP`, and use the `ccmake` command to edit the YARP CMake options. 
```

In my experience, average superbuild users can be easily confused between CMake options of the superbuild and the one of the subprojects, so it is better to be clear. I also modified the YARP option that has a slightly different spelling.

Reply:
> average superbuild users can be easily confused between CMake options of the superbuild and the one of the subprojects, so it is better to be clear

The options seem to be visible only if toggled advanced mode is ON, do you think it is better to mention also that?

Reply:
I added an "advanced" adjective to the suggest phrase, let's hope that this is enough.

Reply:
ok, thanks @traversaro. I have merged your commit suggestions.